codec/manager.go

```diff
@@ -13,6 +13,8 @@ import (
 )
 
 const (
+	CodecVersionSize = wrappers.ShortLen
```

Comment:
I think this stutters:
```suggestion
	VersionSize = wrappers.ShortLen
```